memgpt/memory.py

```diff
@@ -276,6 +276,7 @@ def summarize_messages(
         llm_config=agent_state.llm_config,
         user_id=agent_state.user_id,
         messages=message_sequence,
+        stream=False,
```

Comment:
@sarahwooders This isn't necessary since the default is `False`, but I put this here to signal to devs that we don't support streaming for the summarization subroutine.